### PR TITLE
Remove duplicate codes in "dragged" method in Brush class

### PR DESCRIPTION
**Pull request by susemeeee**

### CODE REVIEW

File: src/com/thunder_cut/graphics/feature/Brush.java [hunk 1]
```diff
@@ -24,99 +26,38 @@ public void pressed(int xPos, int yPos, CanvasPixelInfo canvasPixelInfo, Color c
...
-                else {
-                    currentX--;
-                }
-            }
-
-            return;
-        }
-
-        double ratio;
+        double ratio = Math.max(deltaX, deltaY) / Math.min(deltaX, deltaY);
```
Comment (Kasania) [inline src/com/thunder_cut/graphics/feature/Brush.java]: Math.max and min returns integer value.    
If you need "double" type, you should cast divisor or dividend to "double".

File: src/com/thunder_cut/graphics/feature/Brush.java [hunk 2]
```diff
@@ -24,99 +26,38 @@ public void pressed(int xPos, int yPos, CanvasPixelInfo canvasPixelInfo, Color c
...
                 count = 0;
                 deltaX = Math.abs(xPos - currentX);
                 deltaY = Math.abs(yPos - currentY);
-                if(deltaX > deltaY) {
-                    ratio = deltaX / deltaY;
-                }
-                else {
-                    ratio = deltaY / deltaX;
-                }
+                ratio = Math.max(deltaX, deltaY) / Math.min(deltaX, deltaY);
```
Comment (Kasania) [inline src/com/thunder_cut/graphics/feature/Brush.java]: Math.max and min returns integer value.    
If you need "double" type, you should cast divisor or dividend to "double".

**Review by Kasania — CHANGES_REQUESTED** [review]

**Comment by susemeeee** [comment]
Change ratio to "int" type.
I don't need "double" type.

File: src/com/thunder_cut/graphics/feature/Brush.java [hunk 3]
```diff
@@ -42,7 +42,8 @@ public void dragged(int xPos, int yPos, CanvasPixelInfo canvasPixelInfo, Color c
             return;
         }
 
-        double ratio = Math.max(deltaX, deltaY) / Math.min(deltaX, deltaY);
+        int ratio = Math.max(deltaX, deltaY) / Math.min(deltaX, deltaY);
+        System.out.println(ratio);
```
Comment (Kasania) [inline src/com/thunder_cut/graphics/feature/Brush.java]: Remove test code.

**Review by Kasania — CHANGES_REQUESTED** [review]

**Review by Kasania — APPROVED** [review]
Not perfect, but better than previous version.

**Pull request merged**